eox_tagging/api/v1/viewset.py

```diff
@@ -1,24 +1,25 @@
 """
 Viewset for Tags.
 """
+from django.conf import settings
 from django_filters import rest_framework as filters
 from rest_framework import viewsets
 from rest_framework.authentication import SessionAuthentication
-from rest_framework_oauth.authentication import OAuth2Authentication
 
 from eox_tagging.api.v1.filters import TagFilter
 from eox_tagging.api.v1.pagination import TagApiPagination
 from eox_tagging.api.v1.permissions import EoxTaggingAPIPermission
 from eox_tagging.api.v1.serializers import TagSerializer
 from eox_tagging.edxapp_accessors import get_site
+from eox_tagging.edxapp_wrappers.bearer_authentication import get_bearer_authentication
 from eox_tagging.models import Tag
 
 
 class TagViewSet(viewsets.ModelViewSet):
     """Viewset for listing and creating Tags."""
 
     serializer_class = TagSerializer
-    authentication_classes = (OAuth2Authentication, SessionAuthentication)
+    authentication_classes = (get_bearer_authentication(), SessionAuthentication)
```

Comment:
You could do something like: https://github.com/eduNEXT/eox-tagging/blob/90be49c39d169cf7be924d0398f2e47506d89996/eox_tagging/edxapp_wrappers/course_overview.py#L14
but instead of using CourseOverview it would be BearerAuthentication = get_bearer_authentication()
and on line 14 it would be: https://github.com/eduNEXT/eox-tagging/blob/b7b0c4e36883206715cf1c8cd7cba0a5be64d675/eox_tagging/edxapp_accessors.py#L10

Reply:
solved